src/components/DiffView/index.d.ts

```diff
@@ -0,0 +1,38 @@
+declare module 'react-diff-view' {
+  export interface HunkInfo {
+    content: string;
+    oldStart: number;
+    newStart: number;
+    oldLines: number;
+    newLines: number;
+    changes: Array<object>;
+    isPlain: boolean;
+  }
+
+  export type Hunks = Array<HunkInfo>;
+
+  export interface DiffInfo {
```

Comment:
We don't make use of most of these properties, so maybe we only need to/should document the ones we actually care about? Those would be `hunks` and `type` in this case.